app.arc

```diff
@@ -174,4 +177,5 @@ mission-cloud-platform  # Custom permissions for deployment on Mission Cloud Pla
 email-outgoing  # Grant the Lambda function permission to send email; add email templates.
 email-incoming  # Enable Lambda handlers for incoming emails
 nasa-gcn/architect-plugin-search  # Add an AWS OpenSearch Serverless collection.
-architect/plugin-lambda-invoker
+nasa-gcn/architect-plugin-dynamodb-local
+nasa-gcn/architect-plugin-dynamodb-local-streams
```

Comment:
Just to be clear, in the end you decided to split this functionality across two plugins?

I only see one of these in package.json.

Reply:
They're both there, but I forgot to make them both devDeps, I will change that

Reply:
I'm still confused about this. The @nasa-gcn/architect-plugin-dynamodb-local plugin actually enabled the table streams using the UpdateTable command. But the actual table streams lambda triggering is in the latter plugin? Why not have the UpdateTable command in the latter plugin? Should these still be two separate plugins?